runhouse/logger.py

```diff
@@ -5,21 +5,21 @@
 def get_logger(name: str = __name__):
     logger = logging.getLogger(name)
 
+    # Clear any existing handlers - avoid duplicate logs and maintain consistent log setup across modules
+    logger.handlers.clear()
```

Comment:
wait won't this clear each time each file calls `get_logger`? so when each file is imported and runs... we'll be running tons of these clear and re-adds

Reply:
yea good call we should kill that